Mathlib/Topology/Algebra/Module/FiniteDimension.lean

```diff
@@ -548,3 +549,20 @@ theorem ContinuousLinearMap.exists_right_inverse_of_surjective [FiniteDimensiona
     ∃ g : F →L[𝕜] E, f.comp g = ContinuousLinearMap.id 𝕜 F :=
   let ⟨g, hg⟩ := (f : E →ₗ[𝕜] F).exists_rightInverse_of_surjective hf
   ⟨LinearMap.toContinuousLinearMap g, ContinuousLinearMap.coe_inj.1 hg⟩
+
+/-- If `K` is a complete field and `L` is a finite dimensional vector space over `K`, and `K` is
+locally compact, then `L` is locally compact.
```

Comment:
L is a lousy name for this vector space, why not call it V? It makes the statement easier to understand. And the point, which you don't mention at all in the docstring, is that V has a topology making it into a topological K-module (which means IsTopologicalAddGroup L and ContinuousSMul K L).

IMO the mathlib way to prove this theorem is to prove that under the assumption that K is complete and L is finite-dimensional, L has the K-module topology. The result then follows because this is the product topology.

Reply:
You might be referring to [a theorem that is in FLT](https://github.com/ImperialCollegeLondon/FLT/blob/6c7548617daa2edd9dafe91d38da3fff6846f0f1/FLT/Mathlib/Topology/Algebra/Module/ModuleTopology.lean#L519), which I unfortunately cannot use here...

Reply:
Maybe I should PR this to mathlib; thanks for pointing this out. I guess it shouldn't hold up this PR.

Reply:
The approach suggested by Kevin makes sense, but mathematically it would end up being precisely the same since the proof that any topology is the module topology in this setting would still go through [LinearEquiv.toContinuousLinearEquiv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Module/FiniteDimension.html#LinearEquiv.toContinuousLinearEquiv) (right?).

Unrelated: Kevin, what would you think of renaming `IsModuleTopology` to `IsFinestModuleTopology`? I think that would fit better with the litterature.